private/remote/temurin_archive.bzl

```diff
@@ -1,59 +1,64 @@
+"temurin archive repository rule"
+
+STATIC_MTREE = """\
+etc/ssl/certs/ time=946684800.0 mode=755 gid=0 uid=0 type=dir
+etc/ssl/certs/java/ time=946684800.0 mode=755 gid=0 uid=0 type=dir
+usr/lib/jvm/ time=946684800.0 mode=755 gid=0 uid=0 type=dir
+# NOTE: cacerts is moved to ./etc/ssl/certs/java/cacerts via the awk mutation hence 
+# a symlink created in the original location for completeness.
+usr/lib/jvm/%s/lib/security/cacerts nlink=0 time=946684800.0 mode=777 gid=0 uid=0 type=link link=/etc/ssl/certs/java/cacerts
+"""
+
+AWK = """\
+{
+    sub("^"  "output/lib/security/cacerts", "./etc/ssl/certs/java/cacerts")
+    sub("^"  "output", "./usr/lib/jvm/%s")
+    sub(/time=[0-9\\.]+/, "time=946684800.0");
+    if ($1 ~ ".*legal/.*" || $1 ~ ".*conf/.*") {    
```

Comment:
This is all to match the previous permissions set by pkg_tar workaround, ideally we'd want to remove these.

Reply:
I don't really see where the executable bits are set? Are they all passed through?

Reply:
yes, we preserve what is in the original executable, with some exceptions (to have no diff)

Reply:
Sorry I actually meant to preserve all executables in this change. Are we intentionally removing executable bits to get no diff?

Reply:
The original archive already sets file permissions for executables correctly, so we just need to preserve those. 

However, for entries in `legal` and `conf`, we set some specific permissions so that they match the previous permissions set by pkg_tar.

Reply:
Ideally at some point we should remove those logic from this file and have a diff, but i'd rather have that diff after we confirm this PR doesn't create problems for us.